pages/linux/tune2fs.md

```diff
@@ -0,0 +1,19 @@
+# tune2fs
+
+> Adjust paramters of an ext2, ext3 or ext4 file system.
```

Comment:
Can this be done while the filesystem is mounted, or does it have to be unmounted first?

Reply:
They all can be done live I believe. I checked all the ones in the tldr.